src/Security/Authentication/test/SharedAuthenticationTests.cs

```diff
@@ -552,4 +545,59 @@ public async Task VerifySchemeDefaults()
         Assert.Equal(HandlerType, scheme.HandlerType);
         Assert.Equal(DisplayName, scheme.DisplayName);
     }
+
```

Comment:
So I think we should add something like `DefaultConfigOverwritesPriorValue` test to capture the one situation where the defaultScheme in config might unexpectedly override any previous explicit configures.  Basically if an app Configures the defaultScheme = "foo" before they call AddAuthentication() with no parameters when there's a "configDefault" specified in config, "foo" will get stomped over

In this case, since options goes in order, it will be set to "foo", and then "configDefault".  This seems like reasonable behavior, but might be worth explicitly defining that behavior in a test.

Reply:
If we don't want that behavior, we can write the default config option to only set defaultConfig if its null/empty string, which also seems reasonable

Reply:
Agree with this as well. Not overriding explicitly configured options is the most sensible/predictable strategy here.